subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/alignment/ForcingPlatformAlignmentTest.groovy

```diff
@@ -541,11 +541,11 @@ include 'other'
         ].permutations()*.join("\n")
     }
 
-    @Unroll
     @RequiredFeatures([
         @RequiredFeature(feature = GradleMetadataResolveRunner.REPOSITORY_TYPE, value = "maven"),
         @RequiredFeature(feature = GradleMetadataResolveRunner.EXPERIMENTAL_RESOLVE_BEHAVIOR, value = "true")
     ])
+    @Unroll("can force a published platform version by forcing the platform itself via a constraint")
```

Comment:
What is the benefit of adding a description when it simply repeats the test name?

Reply:
Because of an issue with the way the unroll annotation works with our own runner, if you don't do this you cannot see what repository/experimental combination is running.

Reply:
I was not aware of that, thanks for clarifying.